lib/time.js

```diff
@@ -156,13 +156,18 @@ function CronTime(luxon) {
 					this.utcOffset >= 60 || this.utcOffset <= -60
 						? this.utcOffset / 60
 						: this.utcOffset;
+				
+						// remainder of division by 60
+				let remainder = offset - Math.floor(offset);
+				const remainderMins = remainder * 60;
```

Comment:
rather than use a decimal fraction and multiply it by 60, since we already have access to the minutes with `this.utcOffset`, could you use that to calculate the remaining minutes? like `remainderMins = this.utcOffset - (offset * 60)`? just to minimize the risk of JS weirdness with decimal fractions

while you're there feel free to use `Math.floor(offset)` instead of `parseInt(offset)` and move it above the code you're adding